index.src.html

```diff
@@ -683,7 +688,7 @@ <h4 id="algorithm-require-mediation" algorithm>Require User Mediation</h4>
             // Fetch the form's action URL, passing that new credential object in
             // as the fetch's credentials. If the response indicates success, ask
             // the user agent to ask the user to store the credential for future use:
-            var init = { method: "POST", credentials: c };
+            var init = { method: "POST", body: e.target };
```

Comment:
I have added the `new FormData(e.target)`.

The sign-in case was already up to date by constructing a FormData object manually.

I have left the construction of `PasswordCredential` in line 686 so that a failure to construct the object results in an exception before the login happens.

I have rephrased line 660.